scripts/pi-hole/php/header.php

```diff
@@ -471,29 +471,29 @@ function pidofFTL()
                     </li>
                     <li>
                         <a href="#" id="pihole-disable-10s">
-                            <i class="fa fa-clock-o"></i> <span>For 10 seconds</span>
+                            <i class="fa fa-clock-o"></i> <span>10 Seconds</span>
                         </a>
                     </li>
                     <li>
                         <a href="#" id="pihole-disable-30s">
-                            <i class="fa fa-clock-o"></i> <span>For 30 seconds</span>
+                            <i class="fa fa-clock-o"></i> <span>30 Seconds</span>
                         </a>
                     </li>
                     <li>
                         <a href="#" id="pihole-disable-5m">
-                            <i class="fa fa-clock-o"></i> <span>For 5 minutes</span>
+                            <i class="fa fa-clock-o"></i> <span>5 Minutes</span>
                         </a>
                     </li>
                     <li>
                       <a href="#" id="pihole-disable-cst" data-toggle="modal" data-target="#customDisableModal">
-                            <i class="fa fa-clock-o"></i> <span>Custom time</span>
+                            <i class="fa fa-clock-o"></i> <span>Custom Time</span>
                       </a>
                     </li>
                   </ul>
                     <!-- <a href="#" id="flip-status"><i class="fa fa-stop"></i> <span>Disable</span></a> -->
                 </li>
                 <li id="pihole-enable" class="treeview"<?php if ($pistatus == "1") { ?> hidden="true"<?php } ?>>
-                    <a href="#"><i class="fa fa-play"></i> <span id="enableLabel">Enable&nbsp;&nbsp;&nbsp;<span id="flip-status-enable"></span></span></a>
+                    <a href="#"><i class="fa fa-play"></i> <span id="enableLabel">Enable</span>&nbsp;&nbsp;&nbsp;<span id="flip-status-enable"></span></a>
```

Comment:
This span layout is wanted, as it was part of this fix: https://github.com/pi-hole/AdminLTE/pull/699.
This changes reverts that fix.

Reply:
Ah. Gotit. :)